docs/swagger/spec.yml

```diff
@@ -921,17 +1232,14 @@ definitions:
     type: object
     allOf:
       - $ref: '#/definitions/BaseModel'
+      - $ref: '#/definitions/TimeModelMixin'
     required:
       - name
       - id
     properties:
       name:
         type: string
         description: Display name for organization
-      id:
-        type: string
-        description: UUID for the organization
-        readOnly: true
   UserRole:
```

Comment:
role attribute description should list valid values for "role" similar to the visibility attribute on Buckets.  Should also probably be an enum.